starport_template/lib/widgets/app_version_text.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:cosmos_utils/app_info_extractor.dart';
+import 'package:flutter/foundation.dart';
+import 'package:flutter/material.dart';
+
+/// TODO: Make this intelligent enough to display all of the properties inside [AppInfo] configurable from outside
```

Comment:
could we work on this todo? We could add `AppVersionProvider` to cosmosUtils and pass it as dependency to this widget

Reply:
let's remove this todo now